bgp/modules/terms.py

```diff
@@ -68,24 +64,27 @@ def results(self):
 class ReadingLevelModule:
 
     def __init__(self):
-        self.isbn= None
         self.lexile_min_age= 'None'
         self.lexile_max_age= 'None'
         self.readability_fk_score= None
         self.readability_s_score= None
         self.time = 0
 
-    def run(self, doc, **kwargs):
+    def run(self, book, **kwargs):
+        doc = book.plaintext
+        isbn = 'isbn' in book.metadata and book.metadata['isbn'][0]
+
+        url = 'https://atlas-fab.lexile.com/free/books/' + str(isbn)
```

Comment:
If `'isbn'` is not in `book.metadata` then `isbn` will be `None` and the next line `str(None)` will pass in `'None'`